File: ajax/req.php
<?php
require_once 'HTTP/Request2.php';
$request = new HTTP_Request2();
$request->setUrl('https://db.leyaoyao.com/lyy/rest/group/distributor/login');
$request->setMethod(HTTP_Request2::METHOD_POST);
$request->setConfig(array(
  'follow_redirects' => TRUE
));
$request->setHeader(array(
  'Content-Type' => 'application/x-www-form-urlencoded',
  'Cookie' => 'SESSION=d3899bbe-8cc5-404f-837a-96a796b76ad7'
));
$request->addPostParameter(array(
  'userName' => '13168341703',
  'password' => 'e10adc3949ba59abbe56e057f20f883e'
));
try {
  $response = $request->send();
  if ($response->getStatus() == 200) {
    echo $response->getBody();
  }
  else {
    echo 'Unexpected HTTP status: ' . $response->getStatus() . ' ' .
    $response->getReasonPhrase();
  }
}
catch(HTTP_Request2_Exception $e) {
  echo 'Error: ' . $e->getMessage();
}

File: ajax/req.py
import requests

url = "https://db.leyaoyao.com/lyy/rest/group/distributor/login"

payload = 'userName=13168341703&password=e10adc3949ba59abbe56e057f20f883e'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
  'Cookie': 'SESSION=d3899bbe-8cc5-404f-837a-96a796b76ad7'
}

response = requests.request("POST", url, headers=headers, data = payload)

print(response.text.encode('utf8'))
